README.md

```diff
@@ -1,6 +1,6 @@
 !["Broken" badge to remind us to fix the URLs on the "real" badges](https://img.shields.io/badge/FIX_BADGES-Badges_below_need_to_be_updated-red)
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-3-green.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
```

Comment:
Do we know if they ever use a color other than orange here? I found that orange looked a little "warning – danger" to me, so I just switched to green. I don't really _mind_ orange, but in the context of our other uses of badges it does tend to look like a "bad" thing.